transmart-schemas/src/main/resources/db/changelog/changes/20190617133319_i2b2demodata_change_visit_pk.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.1" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="gijs@thehyve.nl" id="20190617133319-0">
```

Comment:
observation_fact.encounter_num type change missing (numeric to integer)

Reply:
same for encounter_mapping.encounter_num

Reply:
observation_fact.encounter_num already is integer. For encounter_mapping I will change it.